index.html

```diff
@@ -5733,51 +5744,73 @@ <h3>Thing Description Ontology</h3>
-                the client has no prior knowledge of the meaning of <code>status</code>,
-                it must infer possible operations from the semantic annotations available in the TD
-                and its domain-specific ontology, that includes SAREF. The following ontological
-                axioms can be formulated for SAREF, in the OWL Manchester syntax
-                [[owl2-manchester-syntax]]:
+                Given the TD of <a href="#saref-state-annotation-example"></a>, it is straightforward
+                for a client to infer what to do from the internal state description embedded in the TD.
+                However, provided the rules (or axioms) given above, it can be inferred that for all possible
+                models of the light switch described in the TD, an <a>Action</a> may also have the desired effect.
+                A client can therefore adaptively choose to operate on either affordance of the light switch
+                to perform the task. In fact, its behavoir would be unaltered even if the TD did not
+                exposed its state. A single type statement including <code>saref:LightSwitch</code> would
+                suffice, since all light switches behave the same way.
             </p>
 
-            <aside class="example">
-<pre class="nohighlight">
-Class: LightSwitch
-SubClassOf: hasState some OnOffState
-
-Class: ToggleCommand
-SubClassOf: actsUpon some OnOffState</pre>
-            </aside>
-
             <p>
-                They respectively state that a light switch must have an on/off state and that
-                a toggle command must act on some on/off state. They may seem intuitive
-                to humans but it is only possible for the client to know what to do from such
-                axioms. If we assume that a <code>saref:GetCommand</code>
-                always exposes the state of the <code>Thing</code>, the client can then
-                automatically infer what form(s) to submit to change the switch state: a
-                <code>writeproperty</code> on the <code>saref:GetCommand</code> or an
-                <code>invokeaction</code> on the <code>saref:ToggleCommand</code>. Note that the
-                result of the operation cannot be known for certain, as the TD does not expose
-                its internal state. However, by selecting certain models of the physical world,
+                Note that, in the case the internal state is hidden, the result of the operation cannot
+                be known for certain. However, by selecting certain models of the physical world,
                 either by statistical means or by further axiomatization, clients can still
-                reach a certain level of autonomy with an ontological approach.
+                reach a certain level of autonomy with an ontological approach and e.g. learn from their
+                own behavior via a trial and error procedure.
```

Comment:
Is it the client's behavior or the behavior of the thing?
I'm not sure a "trial and error" procedure is practical in the real world.

Reply:
I am indeed referring to the behavior of the client, that learns by interacting with Things. I realize that according to the WoT Architecture terminology, it should be called a [consumer](https://www.w3.org/TR/wot-architecture/#dfn-consumer) instead of a client. I can update the text.

Reply:
See 21221db2a992ed592b615f57baf5e697892183b3.